Fix scan percentage of completion

AlessandroCotto opened the pull request:
This commit add the new endpoint 
```
/burp/scanner/status/percentage
``` 
to retrieve the current scan percentage of completion.

This PR is related to #140 
There was a change in the response of the endpoint 
```
burp/scanner/status
``` 

(For future reference) Here's what I found 
<img width="1123" alt="BurpRestApi_scanStatus" src="https://user-images.githubusercontent.com/5954741/173049822-041b8899-7ba4-409f-a31b-1c8ce3822573.png">

ikkisoft commented:
@AlessandroCotto why not using the new implementation for `/scanner/status`?  If the implementation needs to be different for Burp 1.7 and 2.x, we can determine the version and simply use either one or the other.

I feel that it's confusing to introduce another endpoint for basically the same functionality

AlessandroCotto commented:
I added a new endpoint to be consistent with what I found in the repository. This commit (https://github.com/vmware/burp-rest-api/commit/35468203b0d82502a7ee8b437e7258cdd451a989#diff-f55b524aca8f9e5a659b3c59d4ac2c2cf271f4380d20be58573fe2aaddf297a0) changes the response of the API /scanner/status/ from  ScanProgress to ScanStatusList. 
So I added the new endpoint to restore the functionality and  return the percentage of the scan.
So the two endpoints give related but different informations.

vinilnarayan commented:
Hi Team,
I have clone the recent repo and build the jar with **./gradlew clean build**.
I got the jar named **burp-rest-api-2.1.0.jar** in **burp-rest-api/build/libs** folder.
I replaced the old burp-rest-api jar with new and start burp with the help of **./burp-rest-api.sh**.
The burp is not started. Please find the attached log.

[burp-rest-api.log](https://github.com/vmware/burp-rest-api/files/9052280/burp-rest-api.log)


_The environment where I build burp-rest-api jar_
Java Version : 11.0.11
OS : Mac OS

_The environment where I tried to start burp_
Java Version : 11.0.11
OS : Kali Linux

ikkisoft commented:
There is something wrong the position of the JARs. Have you followed https://github.com/vmware/burp-rest-api#getting-started-or-how-to-run-burp-rest-api step by step? Also, please use `burp-rest-api.sh` from master and not from the release page

vinilnarayan commented:
Hi @ikkisoft,
As for the first step in https://github.com/vmware/burp-rest-api#getting-started-or-how-to-run-burp-rest-api, I checked with release page which have latest released item on Apr 15, 2020. 

Previously I followed the same steps, and that was working perfectly. Now I need the new change for status of scan. for that I tried to build with the repo.

vinilnarayan commented:
When I can expect new release in https://github.com/vmware/burp-rest-api/releases ?

ikkisoft commented:
Today
